### PR TITLE
Add serial dev tip

### PR DESCRIPTION
**Pull request by beaufortfrancois**
Changes proposed in this pull request:

- Adds a new "Dev Tips" section to the Serial API section 
- Mention the internal page to debug Serial. 

FYI @reillyeon

Context: https://chromiumdash.appspot.com/commit/4f1bb9647664c4dd37d165d38b2161f1b121fb18

Live preview: https://deploy-preview-3975--web-dev-staging.netlify.app/serial#dev-tips

### CODE REVIEW

File: src/site/content/en/blog/serial/index.md [hunk 1]
```diff
@@ -505,6 +505,20 @@ const [appReadable, devReadable] = port.readable.tee();
 // and log incoming data in JS console for inspection from devReadable.
 ```
 
+## Dev Tips {: #dev-tips }
+
+Debugging Serial in Chrome is easy with the internal page, `chrome://device-log`
```
Comment (jpmedley) [inline src/site/content/en/blog/serial/index.md:510]:
```suggestion
Debugging the Serial API in Chrome is easy with the internal page, `chrome://device-log`
```

File: src/site/content/en/blog/serial/index.md [hunk 2]
```diff
@@ -505,6 +505,20 @@ const [appReadable, devReadable] = port.readable.tee();
...
+where you can see all serial device related events in one single place.
+
+<figure class="w-figure">
+  <img src="./device-log-page-screenshot.jpg" class="w-screenshot" alt="Screenshot of the internal page to debug Serial.">
```
Comment (jpmedley) [inline src/site/content/en/blog/serial/index.md:514]:
```suggestion
  <img src="./device-log-page-screenshot.jpg" class="w-screenshot" alt="Screenshot of the internal page for debugging the Serial API.">
```

File: src/site/content/en/blog/serial/index.md [hunk 3]
```diff
@@ -505,6 +505,20 @@ const [appReadable, devReadable] = port.readable.tee();
...
+  <figcaption class="w-figcaption">Internal page in Chrome to debug Serial.</figcaption>
```
Comment (jpmedley) [inline src/site/content/en/blog/serial/index.md:515]:
```suggestion
  <figcaption class="w-figcaption">Internal page in Chrome for debugging the Serial API.</figcaption>
```

File: src/site/content/en/blog/serial/index.md [hunk 4]
```diff
@@ -505,6 +505,20 @@ const [appReadable, devReadable] = port.readable.tee();
...
+</figure>
+
+{% Aside %}
+Debugging Serial in the internal page is available in Chrome 87 and later.
```
Comment (jpmedley) [inline src/site/content/en/blog/serial/index.md:519]:
```suggestion
The internal page supports debugging the Serial API in Chrome 87 and later.
```

**Review by jpmedley — CHANGES_REQUESTED** [review]

**Comment by beaufortfrancois** [comment]
Thank you @jpmedley!

**Review by reillyeon — APPROVED** [review]

**Review by jpmedley — APPROVED** [review]

**Pull request merged**